ivy/functional/frontends/paddle/math.py

```diff
@@ -271,6 +271,18 @@ def heaviside(x, y, name=None):
     return ivy.heaviside(x, y)
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("int32", "int64", "float32", "float64")}, "paddle"
+)
+@to_ivy_arrays_and_back
+def increment(x, value=1.0, name=None):
```

Comment:
your issue is related to the `increment` method, please remove this `increment` function as it is a separate issue and is currently being worked on.